### ISSUE TITLE
typescript declaration file 

### ISSUE THREAD

**Issue by sujeshthekkepatt** [http-auth/http-auth#87]
I couldn't find any typescript declaration file. any update on that?

**Comment by gevorg** [comment]
Ok

**Comment by lonix1** [comment]
@sujeshthekkepatt / @dtgriscom Did you find declaration file somewhere or write your own? Any info appreciated.

**Comment by sujeshthekkepatt** [comment]
@lonix1 No. We have removed this package from our dependency. Not because of this issue.
